app/journey/Journey.scala

```diff
@@ -70,22 +70,22 @@ final case class Journey(
     maybeBankDetails:          Option[BankDetails]             = None,
     existingDDBanks:           Option[DirectDebitInstructions] = None,
 
-    maybeTaxpayer:              Option[Taxpayer]              = None,
-    maybePaymentToday:          Option[PaymentToday]          = None,
-    maybePaymentTodayAmount:    Option[PaymentTodayAmount]    = None,
-    maybeMonthlyPaymentAmount:  Option[BigDecimal]            = Some(2000), // TODO OPS-9464 Return the default to None. This is temporary so the journey does not break, whilst the affidrabilty pages are introduced
-    maybeIncome:                Option[Income]                = None,
-    maybeSpending:              Option[Spending]              = None,
-    maybePlanAmountSelection:   Option[BigDecimal]            = Some(2000), // TODO OPS-8654 - connect to calculator form, then set default to None: because of chances to they way payment plan options are generated, customer selection should now be based on regular payment amount, not duration
-    maybeCalculatorDuration:    Option[CalculatorDuration]    = None, // TODO OPS-8654 - remove: because of chances to they way payment plan options are generated, customer selection should now be based on regular payment amount, not duration    maybeArrangementDayOfMonth: Option[ArrangementDayOfMonth] = None,
-    maybeArrangementDayOfMonth: Option[ArrangementDayOfMonth] = None,
-    maybeEligibilityStatus:     Option[EligibilityStatus]     = None,
-    debitDate:                  Option[LocalDate]             = None,
-    ddRef:                      Option[String]                = None,
-    maybeSaUtr:                 Option[String]                = None
+    maybeTaxpayer:                   Option[Taxpayer]                   = None,
+    maybePaymentToday:               Option[PaymentToday]               = None,
+    maybePaymentTodayAmount:         Option[PaymentTodayAmount]         = None,
+    maybeMonthlyPaymentAmount:       Option[BigDecimal]                 = Some(2000), // TODO OPS-9464 Return the default to None. This is temporary so the journey does not break, whilst the affidrabilty pages are introduced
+    maybeIncome:                     Option[Income]                     = None,
+    maybeSpending:                   Option[Spending]                   = None,
+    maybeRegularPlanAmountSelection: Option[PlanRegularAmountSelection] = Some(PlanRegularAmountSelection(2000)), // TODO OPS-8654 - connect to calculator form, then set default to None: because of chances to they way payment plan options are generated, customer selection should now be based on regular payment amount, not duration
```

Comment:
Tiny thing, if you want to fix. Match maybe**_RegularPlanAmountSelection_** with Option[_**PlanRegularAmountSelection**_]. Also I wasn't too clear about what this is just by the name. Is it the amount the user selects to pay monthly? Will it be replacing the maybeMonthlyPaymentAmount?